src/unitxt/metric_utils.py

```diff
@@ -320,6 +320,30 @@ def prepare(self):
 )
 
 
+class EvaluationResults(list):
+    @property
+    def score(self):
+        return self[0]["score"]["global"]
+
+    @property
+    def groups(self):
+        if "groups" not in self[0]["score"]:
+            raise ValueError("Groups scores not found try using group_by in the recipe")
```

Comment:
This comment is not clear. We should have a UnitxtError("No groups were defined using 'group_by' in the recipe.  For more information ....)  and point the 
documentation

Reply:
Will do.